spec/acceptance/nodesets/docker/centos-5.yml

```diff
@@ -10,7 +10,7 @@ HOSTS:
     docker_preserve_image: true
     docker_cmd: '["/sbin/init"]'
     docker_image_commands:
-      - 'yum install -y crontabs initscripts iproute openssl sysvinit-tools tar wget which'
+      - 'yum install -y crontabs initscripts iproute openssl sysvinit-tools tar which'
```

Comment:
all those legacy yaml files in the nodesets directory aren't used. They will be dropped in https://github.com/voxpupuli/puppet-pxe/pull/62. please don't update them